src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -429,14 +411,12 @@ private static String handleNewLine(String str) {
     }
 
     private static boolean checkIfEmptyRow(List<String> rowData) {
-
-        for (String td : rowData) {
-            if (!td.isEmpty()) {
-                return false;
-            }
+        if (rowData.isEmpty()) {
+            return true;
         }
 
-        return true;
+        return rowData.stream()
+                .anyMatch(r -> r.isEmpty());
     }
```

Comment:
Have another look at this one. Doesn't look like the new code is same as previous.

Reply:
@darrenwee This should use `allMatch` and not `anyMatch`. If even a single element is nonempty this method returns false, so this method returns true exactly when all elements are empty.

@damithc Thanks for the catch, I missed this one out.

Reply:
Also, the fact that Travis passes means that we don't have a test for this...

Reply:
Oops, accidentally inverted this in my head. Yes, it should be the complement of `anyMatch`.